.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,21 +11,17 @@ Happy contributing!
 
 ## What does this PR do?
 
-(Provide a description of what this PR does.)
+(Provide a description of what this PR does and why it's needed.)
 
 ## Test Plan
 
-(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)
+(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)
 
 ## Related PRs and Issues
 
-(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)
+- (Related PR or issue)
```

Comment:
```suggestion
- (Related PRs or issues)
```

Reply:
Thanks for the suggestion, but I made it singular because each bullet should be 1 issue or PR.